cmd/sonobuoy/app/gen_config.go

```diff
@@ -14,12 +14,14 @@ import (
 // the default sonobuoy config in a json format.
 func NewCmdGenConfig() *cobra.Command {
 	var f genFlags
+
 	var GenCommand = &cobra.Command{
 		Use:   "config",
 		Short: "Generates a sonobuoy config for input to sonobuoy gen or run.",
 		Run:   genConfigCobra(&f),
 		Args:  cobra.ExactArgs(0),
 	}
+	GenCommand.Flags().AddFlagSet(GenFlagSet(&f, EnabledRBACMode))
```

Comment:
Need to init the flags properly. Has the benefit of having all the relevant gen flags available to make a slightly customized config file. Really not different than even just running gen command and grepping for the config. Not sure if I could write that in fewer LOC though.